src/toast/ops/totalconvolve.py

```diff
@@ -527,15 +527,44 @@ def get_buffer(self, theta, phi, psi, det, verbose):
             timer.report_clear(f"pack input array for detector {det}")
         return pnt
 
-    def conv_and_interpol(self, skycomp, beamcomp, lmax, mmax, pnt, nthreads):
+    # simple approach when there is only one task
+    def conv_and_interpol_scalar(self, skycomp, beamcomp, lmax, mmax, pnt, nthreads):
```

Comment:
Presumably this should be serial rather than scalar?

Reply:
Yeah, that makes more sense. Thanks!